README.md

```diff
@@ -48,20 +46,24 @@ You can make the script run on every boot with either of these methods.
 
 ### Advanced (Administrator method)
 
-In Windows OS, some windows (such as the ones you've run with Administrator privileges, also terminals) the hotkeys will only work if the script itself is `Run as Administrator`, due to the way Windows is designed.
+Windows prevents hotkeys from working in windows that were launched with higher elevation than the AutoHotKey script (such as CMD or Powershell terminals that were launched as Administrator). As a result, Windows Desktop Switcher hotkeys will only work within these windows if the script itself is `Run as Administrator`, due to the way Windows is designed. 
 
-1. In the script folder, create file `desktop_switcher_admin.vbs` with contents
+You can do this by creating a scheduled task to invoke the script at logon. You may use 'Task Scheduler', or create the task in powershell as demonstrated.
 ```
-Set WshShell = CreateObject("WScript.Shell" ) 
-WshShell.Run """desktop_switcher.ahk""", 0 
-Set WshShell = Nothing
+# Run the following commands in an Administrator powershell prompt. 
+# Be sure to specify the correct path to your desktop_switcher.ahk file. 
+
+$A = New-ScheduledTaskAction -Execute "PATH\TO\desktop_switcher.ahk"
+$T = New-ScheduledTaskTrigger -AtLogon
+$P = New-ScheduledTaskPrincipal -GroupId "BUILTIN\Administrators" -RunLevel Highest
+$S = New-ScheduledTaskSettingsSet -AllowStartIfOnBatteries -DontStopIfGoingOnBatteries
+$D = New-ScheduledTask -Action $A -Principal $P -Trigger $T -Settings $S
+Register-ScheduledTask WindowsDesktopSwitcher -InputObject $D
```

Comment:
2. Also, since we'll now use `#include`, powershell might have to set task's working directory to the script too (at least that's what I had to do with the Task Scheduler GUI)
![image](https://user-images.githubusercontent.com/4084885/56083062-f5612d00-5e28-11e9-854d-0df958f366aa.png)

However, if it works without it then it's ok to skip

Reply:
It worked just fine for me - if we get a bug report, we could make this change.